SeleniumWrapper.Core/Forms/BaseForm.cs

```diff
@@ -0,0 +1,41 @@
+using NUnit.Framework;
+
+namespace SeleniumWrapper.Core.Forms;
+
+public abstract class BaseForm
+{
+    protected BaseForm(Browser browser)
```

Comment:
wrong, Browser is handled by containers and manages as singleton for thread. Here Form must be responsible for Page definition